apps/e2e/cypress/e2e/api/login-logout.cy.ts

```diff
@@ -0,0 +1,416 @@
+  // Prepare to intercept/detect relevant GraphQL requests.
+  beforeEach(() => {
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "authorizationQuery");
+    });
+  });
+
+  context("log in", () => {
+    // will log in successfully
+    it("can log in", () => {
+      cy.intercept({ pathname: "/auth-callback" }).as("authCallback");
+
+      // start the login process
+      cy.visit("/login");
+      cy.get("input[name=username]").type(testUserSubject);
+      cy.findByRole("button", { name: "Sign-in" }).click();
+
+      // complete login process
+      cy.wait("@authCallback").then((interception) => {
+        const location = Array.isArray(interception.response.headers.location)
+          ? interception.response.headers.location[0]
+          : interception.response.headers.location;
+        const url = new URL(location);
+        cy.wrap(url.searchParams.get("access_token")).as("accessToken");
+      });
+
+      // auth context provider will update itself
+      cy.wait("@gqlauthorizationQueryQuery").then((interception) => {
+        // make sure we get a user ID back
+        expect(interception.response.body)
+          .nested.property("data.myAuth.id")
+          .matches(uuidRegex);
+
+        cy.get("@accessToken").then((accessToken) => {
+          // make sure it uses the access token
+          expect(interception.request.headers["authorization"]).to.eq(
+            `Bearer ${accessToken}`,
+          );
+        });
+      });
+    });
+
+    // If you log in as a deleted user you end up on the "user deleted" page.
+    it("will show a message when logged in as a deleted user", () => {
+      // stub the "user deleted" API response
+      cy.intercept("POST", "/graphql", (req) => {
+        if (hasOperationName(req, "authorizationQuery")) {
+          // Declare the alias from the initial intercept in the beforeEach
+          req.alias = "gqlauthorizationQueryQuery";
+          // Set req.reply to modify the response
+          req.reply({
+            data: { myAuth: null },
+            errors: [
+              {
+                message: `Login as deleted user: ${testUserSubject}`,
+                extensions: {
+                  reason: "user_deleted",
+                },
+              },
+            ],
+          });
+        }
+      });
+
+      // start the login process
+      cy.visit("/login");
+      cy.get("input[name=username]").type(testUserSubject);
+      cy.findByRole("button", { name: "Sign-in" }).click();
+
+      // the auth response will indicate the user was deleted
+
+      // eventually, we should get to the "user deleted page"
+      cy.findByRole("heading", {
+        name: "Account deleted",
+        level: 1,
+      }).should("exist");
+    });
+  });
+
+  context("refresh", () => {
+    // will log in and refresh the token successfully
+    it.only("can refresh the token", () => {
+      cy.intercept({ pathname: "/refresh*" }).as("refresh");
```

Comment:
Accidental `only` ?

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/21150b43-0c42-4c2e-bbb3-e5dd900bd69f)

Reply:
Same of course happens locally

Reply:
Doh!  It's too easy to do that.
[remove .only](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9159/commits/d74cb186d250b5b26e2f192e28578b58bf213136)